src/server.py

```diff
@@ -448,7 +488,13 @@ def play_sound(self, sound_event_id, connection, sender, path, iface,
             self.players[uuid_].connect("error", self.__player_error_cb,
                                         sound_event_id, uuid_,
                                         connection, path, iface)
+            self._apply_on_loops(uuid_, 'pause')
```

Comment:
Just a nit-pick... what do you think about... `self._apply_on_loops('pause', uuid_to_skip=uuid_)`